style.css

```diff
@@ -0,0 +1,31 @@
+.image{
+    float:left;
+    margin: 44px 70px 7px 0;
+}
+.div-flex{
```

Comment:
The class name should be semantic. Instead of '.div-flex' you can write '.section' or '.item'